push_docker.bsh

```diff
@@ -57,7 +57,7 @@ for DOCKER_FILE in "${IMAGES[@]}"; do
     ${CUR_DIR}/build_dockers.bsh ${DOCKER_FILE}
   fi
 
-  $SUDO docker push gitlfs/linux_packages:${IMAGE_NAME}
+  $SUDO docker push gitlfs/build_dockers:${IMAGE_NAME}
```

Comment:
Should this be `build_dockers` or `build-dockers` like above? If the former, do the above references need to be changed?